applications/asset_tracker/src/env_sensors/CMakeLists.txt

```diff
@@ -5,4 +5,31 @@
 #
 
 zephyr_include_directories(.)
-target_sources(app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/env_sensors.c)
+
+if (CONFIG_USE_BME680_BSEC)
+	if(CONFIG_FP_HARDABI)
+		target_sources(app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/bsec.c)
+		set (bsec_dir ${NRF_DIR}/ext/BSEC_1.4.7.4_Generic_Release)
```

Comment:
People are very particular about how they use version control, it might be that they do not want the bsec library inside of NRF_DIR. So we should allow them to specify it's location somehow, either through a Kconfig option, or a CMake variable.

Reply:
ok. That makes sense.

Reply:
Added a kConfig.